www/js/metrics/ChangeIndicator.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {useMemo} from 'react';
+import { View } from 'react-native';
+import { useTheme, Text } from "react-native-paper";
+import { useTranslation } from 'react-i18next';
+import colorLib from "color";
+
+type Props = {
+    change: {low: number, high: number},
+}
+
+const ChangeIndicator = ({ change }) => {
+    const { colors } = useTheme();
+    const { t } = useTranslation();
+
+    const changeSign = function(changeNum) {
+        if(changeNum > 0) {
+            return "+";
+        } else {
+            return "-";
+        }
+    };
+
+    const changeText = useMemo(() => {
+        if(change) {
+            let low = isNaN(change.low) ? '∞' : Math.round(Math.abs(change.low));
+            let high = isNaN(change.high) ? '∞' : Math.round(Math.abs(change.high));
```

Comment:
do we have screenshots  of what this would look like?  I want to see if `∞` is the best representation, or whether we should use `-` or something else

Reply:
This is something we may want to address before a prod release.
I have not actually seen `∞` get displayed but I have seen the word `infinity` show up if the data is weird. So I don't know if this is really working. We should test and investigate

TODO now

Reply:
Investigating now, and it definitely isn't set up right, I think I need to work on `calculatePercentChange` because it sometimes gives the string "infinity"

Reply:
I found a fix - use `isFinite` instead of `isNaN` -- this will catch "Infinity" and "NaN", when I used {0,0} -> {0,100} as my two weeks low/high values, I was able to have this display: 

![Screenshot 2023-09-25 at 2 33 54 PM](https://github.com/e-mission/e-mission-phone/assets/54848919/f0d86c25-43eb-4d8c-9075-ad09e6de9913)

 If we like this behavior and '∞' I can push this change.

Reply:
Definitely looks better this way!

I think if both boundaries are infinity though, we could consider not showing the change indicator at all, since it's not going to have any useful information

Reply:
I think, in this case, the difference should really be 0 -> infinity, so maybe we should try and represent that? Or just omit it altogether.

Reply:
I have it set up now so that if the range is -inf to +inf, it doesn't show up, because as you said that doesn't mean much. However, if the change is infinite and we're sure of that, show that (ex. 0 emissions to 100 emissions would be infinite change)

Reply:
I think we may want to skip the %  change display in the zero -> 100 case as well because people don't expect to encounter inf in the real world (even if it is mathematically correct). But you can clean that up in TODO